website/docs/layouts/app-frame/partials/code/how-to-use.md

```diff
@@ -1,6 +1,12 @@
 ## How to use this component
 
-The `AppFrame` is a pure layout component that can be used to build the top-level “frame“ of an application. The frame's child containers (“header”, “sidebar“, “footer“, etc.) are agnostic of the content, and don‘t have intrinsic sizes (apart from the ones that are required to make it work as top-level application frame).
+The `AppFrame` is a pure layout component that can be used to build the top-level “frame” of an application. The frame’s child containers (“header”, “sidebar”, “main”, “footer”) are agnostic of the content, and don’t have intrinsic sizes (apart from the ones that are required to make it work as top-level application frame).
+
+### “Main” container
+
+The `AppFrame::Main` child component includes a default `id` with the value `hds-main` on the HTML `<main>` element it renders. This serves as a target for the `hasA11yRefocus` feature skip links which are built into the [`AppHeader`](/components/app-header?tab=code#appheader) and the standalone [`SideNav`](/components/app-header?tab=code#appheader) components.
```

Comment:
Sorry, I missed this in the previous review – I believe the `SideNav` link should look like this, right?
```suggestion
The `AppFrame::Main` child component includes a default `id` with the value `hds-main` on the HTML `<main>` element it renders. This serves as a target for the `hasA11yRefocus` feature skip links which are built into the [`AppHeader`](/components/app-header?tab=code#appheader) and the standalone [`SideNav`](/side-nav?tab=code#side-nav) components.
```

Reply:
fixed locally